Makefile

```diff
@@ -1,6 +1,8 @@
 SHELL = /bin/bash
+TAG = $(shell git describe --exact-match)
```

Comment:
Read the annotated tag from git. Requires the tag to be at current commit. Error otherwise.